### PR TITLE
Implement screen ui

### PR DESCRIPTION
**Pull request by manolobevia**
Use susy 2 for layout, but this will change after using CSS grid layout module

### CODE REVIEW

File: components/oku-circ.vue [hunk 1]
```diff
@@ -1,40 +1,52 @@
...
+    <section class="device-type">
+
+      <h2 class="device-type__header"><i class="fa fa-battery-quarter device-type__header--icon-charger" aria-hidden="true"></i> Phone Chargers: </h2>
 
       <ul>
-        <li><span>{{ phoneChargers[location].iphone4Available }}</span> iPhone 4 available</li>
-        <li><span>{{ phoneChargers[location].iphoneAvailable}}</span> iPhone 5 &amp; up available</li>
-        <li><span>{{ phoneChargers[location].microUsbAvailable }}</span> Micro USB available</li>
-        <li><span>{{ phoneChargers[location].usbCAvailable }}</span> Micro USB-C available</li>
+        <li class="device-type__info">
```
Comment (cappadona) [inline components/oku-circ.vue]: ripe for DRYing up via loop, but we can save that for later improvement

File: components/oku-circ.vue [hunk 2]
```diff
@@ -1,40 +1,52 @@
...
+
+        <li class="device-type__info"><span class="device-type__info--count">{{ phoneChargers[location].usbCAvailable }}</span> USB-C available
+        </li>
       </ul>
     </section>
   </div>
 </template>
 
 <script>
-import _ from 'lodash'
```
Comment (cappadona) [inline components/oku-circ.vue]: Well played 😄

File: components/oku-circ.vue [hunk 3]
```diff
@@ -49,5 +61,130 @@ export default {
 }
 </script>
 
-<style>
+<style lang="scss">
+
+  @import '~susy';
+  @import '~font-awesome/css/font-awesome.min.css';
```
Comment (cappadona) [inline components/oku-circ.vue]: Once Nuxt `v1.0` is released, we should be able to switch over to the [Font Awesome module](https://github.com/nuxt-community/modules/tree/master/modules/font-awesome)

File: package.json [hunk 4]
```diff
@@ -8,10 +8,13 @@
   ],
   "private": true,
   "dependencies": {
+    "add": "^2.0.6",
     "axios": "^0.16.1",
     "lodash": "^4.17.4",
     "nuxt": "latest",
-    "nuxt-helpers": "^0.10.0"
+    "nuxt-helpers": "^0.10.0",
+    "susy": "^2.2.12",
```
Comment (cappadona) [inline package.json]: Susy should be a `devDependency`

File: package.json [hunk 5]
```diff
@@ -8,10 +8,13 @@
...
+    "yarn": "^0.24.5"
```
Comment (cappadona) [inline package.json]: It's not necessary to include Yarn as a dependency

**Comment by manolobevia** [comment]
I have removed yarn as dependency, and added susy as dev dependency.

**Pull request merged**